File: src/level1/SumBetweenTwoInt.go
func solution(a int, b int) int64 {
    sum := 0
    if a > b {
        for i := b; i <=a; i++{
            sum += i
        }
    }else if a < b{
        for i := a; i <=b; i++{
            sum += i
        }
    }else{
        sum = a
    }
    var i64 int64
    i64 = int64(sum)
    return i64
}

File: src/level1/SumBetweenTwoInt.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        if(a > b){
			//answer += b;
			for(int i = b; i <= a; i++){
				answer += i;
				
			}
		}
		if(a < b){
            //answer += a;
			for(int i = a; i <= b; i++){
				answer += i;
			}
		}
        if(a == b){
            answer = a;
        }
        return answer;
    }
}